doc/benchmarking.md

```diff
@@ -36,6 +36,14 @@ The output will look similar to:
 ...
 ```
 
+Some benchmarks create a test data directory within
+`test_common_Bitcoin Core/`, which is within the system's temporary directory
+(for example, `/tmp`). By default the directory is
+named as a random string, but this can be
+overridden with the `BITCOIN_TEST_PATH` environment variable.
+Be careful not to run multiple simultaneous instances of `bench_bitcoin`
+that share the same test data directory; they'll conflict with each other.
```

Comment:
What about creating a pid file during init and removing it during shutdown? Failing early if the file already exist.

Reply:
Yes, perhaps even better, maybe the test startup code can call `LockDirectory()` like `bitcoind` does. I'll see if I can get that to work, thanks for the suggestion!